Tasks/Common/npm-common/Strings/resources.resjson/de-de/resources.resjson

```diff
@@ -0,0 +1,17 @@
+{
+  "loc.friendlyName": "npm",
+  "loc.helpMarkDown": "[Weitere Informationen](https://go.microsoft.com/fwlink/?LinkID=613746)",
+  "loc.description": "Führt einen npm-Befehl aus.",
+  "loc.instanceNameFormat": "npm $(command)",
+  "loc.input.label.cwd": "Arbeitsordner",
+  "loc.input.help.cwd": "Das Arbeitsverzeichnis, in dem der npm-Befehl ausgeführt wird. Der Wert wird standardmäßig auf den Stamm des Repositorys festgelegt.",
+  "loc.input.label.command": "npm-Befehl",
+  "loc.input.label.arguments": "Argumente",
+  "loc.input.help.arguments": "Weitere Argumente, die an npm übergeben werden.",
+  "loc.messages.InvalidCommand": "Für npm sollte nur ein Befehl verwendet werden. Verwenden Sie die Eingabe \"Argumente\" für zusätzliche Argumente.",
+  "loc.messages.NpmReturnCode": "Npm wurde mit dem folgenden Rückgabecode beendet: %d",
+  "loc.messages.NpmFailed": "Npm-Fehler: %s",
```

Comment:
I don't believe that any of these loc files will be loaded at runtime, as the tasks are only loading the task resources.
For NuGet, we copy the common strings to all tasks that will need them.

Reply:
Removed.